src/styles/blocks/main.scss

```diff
@@ -0,0 +1,28 @@
+main {
+  padding: 50px 40px;
+
+  --column-counts: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--column-counts), 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
```

Comment:
Make sure to use kebab-case for CSS custom properties (variables). Instead of '--column-counts', it should be '--column-counts'.